org-code-javabuilder/neighborhood/src/main/java/org/code/neighborhood/Grid.java

```diff
@@ -41,12 +42,16 @@ public GridSquare getSquare(int x, int y) {
 
   // Hides all buckets from the screen
   public void hideBuckets() {
-    System.out.println("You hid the buckets");
+    NeighborhoodOutputHandler.sendMessage(
+            new NeighborhoodSignalMessage(NeighborhoodSignalKey.HIDE_BUCKETS, new HashMap<>())
+    );
   }
 
   // Displays all buckets on the screen
   public void showBuckets() {
-    System.out.println("You displayed the buckets");
+    NeighborhoodOutputHandler.sendMessage(
```

Comment:
[Non-blocking comment] I think Hannah wanted students to be able to call this directly from the Painter object. We'll want to double check the use case. Definitely not blocking for this PR as that could take a couple cycles to figure out.